login_ui.py

```diff
@@ -58,5 +59,6 @@ def writeLogins(self):
 if __name__ == '__main__':
 
     app = QtWidgets.QApplication(sys.argv)
-    ex = Example()
-    sys.exit(app.exec_())
\ No newline at end of file
+    ex = AuthWindow()
+    sys.exit(app.exec_())
+    
```

Comment:
in python there is convention that last line suppose to be empty

Reply:
done